src/test/groovy/org/prebid/server/functional/tests/privacy/GppTransmitTidActivitiesSpec.groovy

```diff
@@ -604,15 +646,21 @@ class GppTransmitTidActivitiesSpec extends PrivacyBaseSpec {
             !bidderRequest.imp[0].ext.tid
             !bidderRequest.source.tid
         }
+
+        and: "Don’t affect source.schain if it was present"
+        assert bidderRequest.source.schain == sourceWithSchainAndTid.schain
     }
 
-    def "PBS amp should remove bidRequest.(source/imp[0].ext).tid when ext.prebid.createTids=null and defaultAction=false"() {
+    def "PBS amp should remove bidRequest.(source/imp[0].ext).tid but don't change schain in request when ext.prebid.createTids=#createTid and defaultAction=false"() {
```

Comment:
`#createTid` still static, meaning `null`
`PBS amp should remove bidRequest.(source/imp[0].ext).tid and don't change schain in request when ext.prebid.createTids=#createTid and defaultAction=false and schain specified in request`

Reply:
updated